crates/common/src/lib.rs

```diff
@@ -20,7 +20,7 @@ pub use header::{BlockHeader, BlockHeaderBuilder};
 
 /// The address of a Starknet contract.
 #[derive(Copy, Clone, PartialEq, Eq, Hash, Serialize, PartialOrd, Ord)]
-pub struct ContractAddress(Felt);
+pub struct ContractAddress(pub Felt);
```

Comment:
The reason this `Felt` is private is because it has an additional invariant that needs upholding. More specifically it must be `< 2^251` whereas a normal felt can be a little bigger than that.

To work-around this we have fallible constructor `ContractAddress::new` or `ContractAddress::new_or_panic` for tests.

Its the `macros::starkhash251::newtype!` types that have this restriction in place. We should probably refactor these to have a `Felt251` inner type instead.